mne/io/boxy/boxy.py

```diff
@@ -0,0 +1,404 @@
+# Authors: Kyle Mathewson, Jonathan Kuziek <kuziekj@ualberta.ca>
+#
+# License: BSD (3-clause)
+
+from configparser import ConfigParser, RawConfigParser
+import glob as glob
+import re as re
+import os.path as op
+import pandas as pd
+import numpy as np
+
+import mne 
```

Comment:
all `mne` imports should be relative not absolute (start with `from .`)

Reply:
almost got this, but when I try to import some relative functions I get this error, perhaps need to import something else first? 

``` 
File "/Users/kylemathewson/mne-python/mne/io/boxy/boxy.py", line 19, in <module>
    from ...coreg import coregister_fiducials
  File "/Users/kylemathewson/mne-python/mne/coreg.py", line 23, in <module>
    from .label import read_label, Label
  File "/Users/kylemathewson/mne-python/mne/label.py", line 18, in <module>
    from .source_estimate import (SourceEstimate, _center_of_mass,
  File "/Users/kylemathewson/mne-python/mne/source_estimate.py", line 17, in <module>
    from .cov import Covariance
  File "/Users/kylemathewson/mne-python/mne/cov.py", line 20, in <module>
    from .io import fiff_open, RawArray
ImportError: cannot import name 'RawArray' from 'mne.io' (/Users/kylemathewson/mne-python/mne/io/__init__.py)
```

Reply:
if the file is `mne/io/boxy/boxy.py` then I think you'd need to go up two levels, i.e., `from ..io import RawArray` (with two leading dots)

Reply:
@drammock That is my confusion,  I'm not even trying to import RawArray, it is getting imported by cov.py, which is imported by source_estimate.py, which is imported by label.py, which is imported by coreg.py when I try to import coregister_fiducials

Reply:
ah, I didn't read the traceback carefully enough. That is indeed puzzling.

Reply:
I can't replicate this.  I created a new file `mne/io/brainvision/foo.py` with the contents:

```
from ...coreg import coregister_fiducials
print(coregister_fiducials)
```

and in a session I'm able to do
```python
from mne.io.brainvision.foo import coregister_fiducials
foo.coregister_fiducials
```
with no errors.

Reply:
foo.py:
```py
from ...coreg import coregister_fiducials
print(coregister_fiducials)
```

```
(mnedev) kyles-mbp:mne-python kylemathewson$ python -i mne/io/brainvision/foo.py
Traceback (most recent call last):
  File "mne/io/brainvision/foo.py", line 1, in <module>
    from ...coreg import coregister_fiducials
```

Reply:
that error is expected if you simply try to run `foo.py` from the terminal; when you run `foo.py` directly it is defined as `__main__` and doesn't know it's part of a larger module.  But if you fire up a session and run 

```python
from mne.io.brainvision.foo import coregister_fiducials
foo.coregister_fiducials
```

as I suggested above, that should work?

Reply:
This sounds like a circular import problem, nest the imports having to do with `.coreg`

Reply:
@drammock yes that does run with without the foo. 
```
from mne.io.brainvision.foo import coregister_fiducials
coregister_fiducials
```

@larsoner - a bit novice in python dev, can you help with an example of a nested import in this case, sorry to be slow

Reply:
@kylemath "nested import" means importing `coregister_fiducials` from inside the class, function, or method definition where it's needed, rather than at the top of the file.

Reply:
(so within the `__init__` of `RawBOXY` presumably)

Reply:
yup, thats it, working now, thanks for the help